src/ppx/grfx/vk/vk_command.cpp

```diff
@@ -335,18 +362,44 @@ void CommandBuffer::ClearDepthStencil(
     const grfx::DepthStencilClearValue& clearValue,
     uint32_t                            clearFlags)
 {
-    auto pCurrentRenderPass = GetCurrentRenderPass();
-    if (IsNull(pCurrentRenderPass)) {
+    if (!HasActiveRenderPass()) {
         return;
     }
 
-    // Make sure pImage is depth stencil in current render pass
-    if (pCurrentRenderPass->GetDepthStencilImage().Get() != pImage) {
-        return;
+    grfx::Rect renderArea;
+    uint32_t   baseArrayLayer;
+
+    // Dynamic render pass
+    if (mDynamicRenderPassActive) {
+        // Clear rect
+        renderArea = mRenderArea;
+
+        // Make sure pImage is depth stencil in depth stencil view
+        if (mDepthStencilView->GetImage().Get() != pImage) {
+            return;
+        }
+
+        // Get array layer
+        baseArrayLayer = mDepthStencilView->GetArrayLayer();
     }
+    else {
+        // active regular render pass
+        auto pCurrentRenderPass = GetCurrentRenderPass();
+
+        // Make sure pImage is depth stencil in current render pass
+        if (pCurrentRenderPass->GetDepthStencilImage().Get() != pImage) {
+            return;
+        }
 
-    // Get view
-    auto view = pCurrentRenderPass->GetDepthStencilView();
+        // Get view
```

Comment:
Google style usually discourage adding comments that simply reformulates the associated line:
```
// get array layer
baseArrayLayer = view->GetArrayLayer()
```
The comment just repeats the function name, without adding value.
A few comments in this PR could be removed.

Reply:
As Nathan mentioned, I think we can remove a few of these: on line 394, 397, 401